common/cmake/devices.cmake

```diff
@@ -1061,13 +1061,15 @@ function(ADD_BITSTREAM_TARGET)
         DEPENDS ${BITSTREAM_DEPS}
         COMMAND
           ${CMAKE_COMMAND} -E make_directory ${OUT_LOCAL}
+        COMMAND rm -f ${OUT_FASM_MERGED}
         COMMAND cat ${ALL_OUT_FASM} >> ${OUT_FASM_MERGED}
```

Comment:
Could just change `>>` to `>` and it would do the same thing without needing the `rm`?

Reply:
Unsure if this would break the multiple fasm files that get output with cat. Most probably you are right and we could just use >

Reply:
It would break things if there were multiple targets that output to the same file, but that strategy would be a bug in the first place because if `--jobs` is greater than 1, then there would be a race condition.

Bottom line, just use `>` and drop the `rm -f`.  Even if `>>` was required, now that there is a `rm -f` first, the `>>` no longer does anything compared with just `>`.

Reply:
Ok done